.env

```diff
@@ -0,0 +1,3 @@
+URL= "https://magento.softwaretestingboard.com/what-is-new.html"
+USERNAME= "safiye.ipek96@gmail.com"
+PASSWORD= "QATesting1!"
```

Comment:
I would personally avoid pushing `.env`file into repository, since it is considered to be sensitive information. I would suggest to add this file to .gitignore
To show which parameters in .env file to provide, usually it is helpful to create `.env.example`file and push to the repository. Provide this information inside of the file:
```
USERNAME= ""
PASSWORD= ""
```
Base URL usually can be stored inside `playwright.config.ts` file though.

Reply:
Thank you Eleonora for the feedback , it is very useful , I just addressed the feedback

Reply:
Thank you for fixing it all and sorry for the super late response (have been on vacation and had a few projects going on at work ...).
I would also exclude .env file and include it into .gitignore, since we don't want to expose these values in repo.